cmake/Modules/FindKIM-API.cmake

```diff
@@ -37,6 +37,26 @@
 # KIM-API-CMAKE_Fortran_COMPILER
 #
 
+function(_KIMAPI_GET_VERSION _OUT_ver _version_hdr)
+  if(NOT EXISTS ${_version_hdr})
+    message(FATAL_ERROR "Header file ${_version_hdr} not found (check value of KIM-API_INCLUDE_DIR)")
+  endif()
+  foreach(_var KIM_VERSION_MAJOR KIM_VERSION_MINOR KIM_VERSION_PATCH)  
+    file(STRINGS ${_version_hdr} _contents REGEX "#define ${_var}[ \t]+")
+    if(_contents)
+      string(REGEX REPLACE ".*#define ${_var}[ \t]+([0-9.]+).*" "\\1" _${_var} "${_contents}")
+      if(NOT ${_${_var}})
```

Comment:
fails if value is `0`.  use `if("${_${_var}}" STREQUAL "")`

Reply:
Done.